iceoryx_meta/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 cmake_minimum_required(VERSION 3.14)
-set(iceoryx_VERSION 0.17.0.2)
-
+file (STRINGS "../VERSION" iceoryx_VERSION)
```

Comment:
I do not like this relative path but I think there is no alternative?! @elBoberido you are our CMake hero! Please help us. do you know stuff?

Reply:
Yeah I also wasn't too happy about it. Would be glad to learn an alternative.

Reply:
I also don't have a better idea. The only other idea I have is to use the VERSION file in iceoryx_utils and iceoryx_posh and use a script to change the version number. But that also no nice solution